pkg/scheduler/apis/config/types.go

```diff
@@ -220,6 +220,15 @@ type PluginConfig struct {
 	Args runtime.Unknown
 }
 
+// NodeLabelPresenceArgs holds the args that are used to configure the corresponding plugin.
```

Comment:
I am hopeful that we can deprecate most of the custom predicates/priorities since there are equivalent features available via the Pod API. Therefore, I think this plugin will be an "internal" plugin that exist just to support the Policy API until it is deprecated, so I don't think we should  allow to configure it via ComponentConfig, and its args should be defined internally in the plugin file itself for example.

Reply:
SGTM. Moved to the plugin.